{{cookiecutter.project_slug}}/cmake/cache.cmake

```diff
@@ -0,0 +1,16 @@
+set(CACHE_OPTION "ccache" CACHE STRING "Compiler cache to be used.")
```

Comment:
Does this code support not using any cache? I'm not sure it does, or how.

Reply:
Nops, I added back the `ENABLE_CACHE` option.